.github/workflows/test.yml

```diff
@@ -126,6 +126,15 @@ jobs:
         TEST_ENV: ${{ steps.set-env-step.outputs.test-env }}
       run: ./gradlew ${{ matrix.gradleTask }} --scan
 
+    - name: Codacy Coverage Reporter
+      if: matrix.gradleTask == 'unitTest'
+      uses: codacy/codacy-coverage-reporter-action@0.2.0
+      env:
+        CODACY_API_BASE_URL: https://app.codacy.com/gh/DataBiosphere/terra-workspace-manager
+      with:
+        project-token: ${{ secrets.CODACY_PROJECT_TOKEN }}
+        coverage-reports: build/reports/jacoco/test/jacoco.xml
+
```

Comment:
I'm not sure what the default behavior is ... if the codacy upload fails, will the entire action fail? Do we want it to? (pro: we notice the problem; con: could interrupt development). Maybe this wants continue-on-error?